Source/WebCore/html/parser/HTMLDocumentParserFastPath.cpp

```diff
@@ -480,21 +480,21 @@ class HTMLFastPathParser {
         auto* cursor = start;
         const auto* end = start + m_parsingBuffer.lengthRemaining();
         ([&]() ALWAYS_INLINE_LAMBDA {
-            constexpr size_t stride = 16 / sizeof(CharacterType);
+            constexpr size_t stride = SIMD::stride<CharacterType>;
             using UnsignedType = std::make_unsigned_t<CharacterType>;
             if (static_cast<size_t>(end - cursor) >= stride) {
-                const auto quoteMask = SIMD::splat(static_cast<UnsignedType>('<'));
-                const auto escapeMask = SIMD::splat(static_cast<UnsignedType>('&'));
-                const auto newlineMask = SIMD::splat(static_cast<UnsignedType>('\r'));
-                const auto zeroMask = SIMD::splat(static_cast<UnsignedType>(0));
+                const auto quoteMask = SIMD::splat<UnsignedType>('<');
```

Comment:
Can these be `constexpr` like is done in the `LiteralParser` for splats?

Reply:
Done.